tests/e2e/support/commands.js

```diff
@@ -411,6 +411,24 @@ Cypress.Commands.add('playPointsSpectator', (card, pNum) => {
     });
 });
 
+Cypress.Commands.add('concedeSpectator', (pNum) => {
+  io.socket.get('/game/concede', { pNum }, function handleResponse(res, jwres) {
+    if (jwres.statusCode !== 200) {
+      throw new Error(jwres.body.message);
+    }
+    return jwres;
+  });
+});
+
+Cypress.Commands.add('rematchSpectator', (pNum) => {
+  io.socket.get('/game/rematch', { pNum }, function handleResponse(res, jwres) {
+    if (jwres.statusCode !== 200) {
+      throw new Error(jwres.body.message);
+    }
+    return jwres;
+  });
+});
```

Comment:
These look at the same as `concedeOpponent()` and `rematchOpponent()`, no? Are they necessary?

Reply:
Good catch these are not needed anymore